compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -233,10 +233,15 @@ class TR_ClobberEvalData
 
 namespace TR
    {
+   enum ExternalRelocationPositionRequest
+      {
+      ExternalRelocationAtFront,
+      ExternalRelocationAtBack,
+      };
    enum AOTRelocationPositionRequest
```

Comment:
Will we be cleaning up the duplicates afterward?

Reply:
Yes, the methods and enums with AOT in their names will be removed once downstream is fixed.